reproject/utils.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-import sys
+from six import string_types
```

Comment:
six is already imported below:

```
 from astropy.extern import six
```

so just use ``six.string_types`` and don't add an import

Reply:
Thank you! I missed that line of code...